models/users.js

```diff
@@ -864,6 +864,21 @@ const getNonNickNameSyncedUsers = async () => {
   }
 };
 
+const fetchArchivedUsers = async () => {
+  try {
+    const archivedUsers = [];
+    const users = await userModel.where("roles.archived", "==", true).get();
+    users.forEach((doc) => {
```

Comment:
Each user is called `doc` here, naming can be improved I think.

Reply:
these are not users data , they are Documet snapshots from firestore ,from which we get data by calling ``doc.data()`` so  i have changed the name accordingly now please check